task_router_queue.go

```diff
@@ -0,0 +1,87 @@
+package twilio
```

Comment:
Can we merge this + activity.go into just `task_router.go` and have all of the logic go there? If that file gets unwieldy we can talk about splitting out again. But that file could also hold `WorkspaceService` which makes sense, I think.

Reply:
Interested to hear your reasoning behind this? I think it might be nicer to keep them in individual files but move to a `task_router` folder?

Reply:
I'm not sure we want to or can split out the dependencies. I guess the idea is to limit the growth of files in this repo & keep like things together, so long as each file is of a reasonable size.

Reply:
I've made the other changes in this PR but not sure if this is a good idea. Adding 4 extra files (plus 4 tests) I think is better than adding everything into one file. If you look through the official Twilio APIs they are separate classes, but I'll leave the decision up to you :)